### ISSUE TITLE
Direct FTP access to ftp://ftp.mozilla.org/ times out

### ISSUE THREAD

**Issue by KaptenJansson** [DamonOehlman/djo-shell#2]
ftp://ftp.mozilla.org/ no longer works while http://ftp.mozilla.org/ does, however the sed -r regexp command is setup to expect the --list-only result from curl, not the contents of a HTML page.

This causes firefox version lookup to fail in https://github.com/DamonOehlman/djo-shell/blob/master/browser-version.sh#L26

@fippo @jan-ivar FYI

**Comment by KaptenJansson** [comment]
FYI for those who don't know this is a dependency for travis-multirunner.

**Comment by hfiguiere** [comment]
FTP protocol has been disabled. Use https and use archive.mozilla.org as the proper host.

**Comment by fippo** [comment]
how about this? I think this works but don't have access to the old format

```
    stable)
      TARGET=http://$BASE_FIREFOX_STABLE`curl -s --list-only https://$BASE_FIREFOX_STABLE/ |grep -e \.tar\.bz2 | sed -r "s/^.*(firefox-[0-9\.ba]+\..*tar.bz2).*/\1/"`
```

**Comment by DamonOehlman** [comment]
Thanks all for highlighting this. Investigating the workaround now - I'm hopeful that @fippo's suggestion just works :)

**Comment by DamonOehlman** [comment]
Published `1.4.1` which implements #3 - should be resolved now.  Thanks @fippo for the fix!
